src/docs/arch-eviction.dox

```diff
@@ -30,8 +30,21 @@ eviction threads.
 
 It is possible to have just the eviction server and no worker threads. In such a case,
 the eviction server walks the tree to find pages to evict and evict these pages from the
-cache. Pages that are being marked for forced eviction go onto the urgent
-queue, and these pages take precedence over the pages from the ordinary eviction queues.
+cache. Pages that are being marked for forced eviction go onto the urgent queue if the following
+conditions are met; otherwise the page can be evicted by the application thread itself:
```

Comment:
Should we remove the "otherwise" part? We cover this line 38.

Reply:
I've removed the repeated part in line 38 as it has the same info but in less detail